v2/manifests/deps/vault.yml

```diff
@@ -9,3 +9,8 @@ services:
     environment:
       VAULT_DEV_ROOT_TOKEN_ID: '0000-0000-0000-0000'
       VAULT_DEV_LISTEN_ADDRESS: '0.0.0.0:8200'
+    healthcheck:
+      test: [ "CMD", "wget", "-q", "-O", "-", "http://localhost:8200/v1/sys/health?drsecondarycode=200&performancestandbycode=200&standbycode=200" ]
```

Comment:
Just querying why `wget` when everything else is using `curl`?

Reply:
This is because the vault docker image we are using doesn't have `curl` installed.